apps/docs/pages/guides/auth/social-login.mdx

```diff
@@ -41,6 +41,12 @@ Supabase supports a suite of social providers. Follow these guides to configure
       </a>
     </Link>
 ))}
+  
+## Provider Tokens
```

Comment:
OK to move this outside of that closing </div> below?

Reply:
Oh my bad. Didn't see the </div>